### PR TITLE
Move Data Hub administration documentation to Control Center section (CAT-1772)

### PR DESCRIPTION
**Pull request by dtitenko**

### CODE REVIEW

**Comment by MariaShaposhnikova** [comment]
@dtitenko, can you sign CLA please?

**Comment by Adam-Dupaski** [comment]
Needs RN on both the Control Center and the Data Hub sides.

**Review by Adam-Dupaski — APPROVED** [review]

File: content/developerportal/control-center/data-hub-admin.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Custom owners are owners that have been added as contacts for a registered app. They may be added during the app [curation](/data-hub/data-hub-catalog/curate#custom-owner), or they may have been specified during app registration. For more information, see the [Adding a Custom Owner](/data-hub/data-hub-catalog/curate#custom-owner) section of *How to Curate Registered Assets*.
+
+Custom owners are only the contact people for registered assets. Adding a custom owner does not give them access rights to the Data Hub Catalog, as they are not able to log in to the Mendix Platform and curate registered assets in the Catalog. Custom owners appear in the Catalog with their name and initials, or with their personalized avatar if they are a Mendix user.
+
+Data Hub Curators and Mendix Admins can manage the custom owner list on the **Owners** tab:
+
+![Owners](attachments/data-hub-admin/owners.png)
+
+{{% alert type="info" %}}
+Only Data Hub Curators see the **Owners** tab.
```
Comment (Adam-Dupaski) [inline content/developerportal/control-center/data-hub-admin.md:49]: @dtitenko can you please verify this note? I reworded it, but not sure if it's technically accurate now.

Comment (dtitenko) [reply content/developerportal/control-center/data-hub-admin.md:49]: Yes, this is not correct. Data Hub Curators will see **only** Owners tab. Mendix Admin will see 3 tabs but a Curator only 1.

Comment (dtitenko) [reply content/developerportal/control-center/data-hub-admin.md:49]: Let me rephrase it later today.

Comment (Adam-Dupaski) [reply content/developerportal/control-center/data-hub-admin.md]: https://github.com/mendix/docs/pull/3610/commits/b5e72cf02a0381c401a8c94da9e1c8170b0c09cf

File: content/developerportal/control-center/data-hub-admin.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+A Mendix Admin sees the **Curators** tab displaying the list of Data Hub Curators for the organization:
+
+![Data Hub Curators](attachments/data-hub-admin/curators.png)
+
+To assign a Curator role to a Mendix user, click **Add Curator**. To search the list of Mendix users in your organization, start typing in the search box and check the user(s) you want to assign the Curator role to.
+
+{{% alert type="info" %}}This list shows all Mendix users for your organizations. It does not include non-Mendix users that are added as [custom owners](#custom-owners).{{% /alert %}}
+
+{{% alert type="info" %}}
```
Comment (ezagroba) [inline content/developerportal/control-center/data-hub-admin.md:28]: You already said all of this, delete this whole paragraph.

File: content/data-hub/index.md [hunk 3]
```diff
@@ -55,26 +55,24 @@ Data Hub users can do the following:
 A user who publishes an OData service is the technical owner of the service and thereby has a [Data Hub Curator](#curator) role over their own services. If they have registered services which are not published and are non-discoverable, only they will be able to find and consume these services. 
 {{% /alert %}}
 
-### 3.2 Data Hub Admin {#admin}
+### 3.2 Mendix Admin {#admin}
 
-The Data Hub Admin role can carry out Data Hub operations and governance in collaboration with the organization's data stewards, security officers, and security architects. 
+The Mendix Admin role can carry out Data Hub operations and governance in collaboration with the organization's data stewards, security officers, and security architects.
```
[the remark this answers is not in the document]
Comment (Adam-Dupaski) [reply content/data-hub/index.md]: Removed: https://github.com/mendix/docs/pull/3610/commits/79f193719f8155c18bc89fce467cbc59e3784013

File: content/developerportal/control-center/data-hub-admin.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+* Assign and manage Data Hub [Curators](#curator)
+* Invite and manage custom [Owners](#custom-owners) (who have already been added as business or technical owners in the Data Hub Catalog)
+* Invite and manage [External Users](#external-users)
+
+## 2 Curators {#curator}
+
+A Data Hub Curator can perform day-to-day management functions on all registered assets in the Data Hub Catalog and also enrich the Catalog information on registered assets that is displayed. Curators can see and curate all registered assets in the Data Hub Catalog. Mendix users who own registered assets can curate their own items, but not those that they do not own.
+
+A Mendix Admin sees the **Curators** tab displaying the list of Data Hub Curators for the organization:
```
[the remark this answers is not in the document]
Comment (Adam-Dupaski) [reply content/developerportal/control-center/data-hub-admin.md:20]: It's assumed this is for Mendix Admins, as they can see all pages of Control Center. But that first sentence describing the Curator role can be moved below the image for clarity.

[the remark this answers is not in the document]
Comment (Adam-Dupaski) [reply content/developerportal/control-center/data-hub-admin.md:18]: Moved: https://github.com/mendix/docs/pull/3610/commits/5e3b7a77afeb40cd52bd9fd24a33d4e1f245d37f

File: content/developerportal/control-center/data-hub-admin.md (hunk 2 above)
Comment (Adam-Dupaski) [reply content/developerportal/control-center/data-hub-admin.md:26]: Removed: https://github.com/mendix/docs/pull/3610/commits/c16f6701582ffc577f0905be0386f472e93e3d02

**Review by Adam-Dupaski — APPROVED** [review]
